code/core/src/manager/components/sidebar/Search.tsx

```diff
@@ -2,13 +2,15 @@ import React, { useCallback, useRef, useState } from 'react';
 
 import { IconButton, TooltipNote, WithTooltip } from '@storybook/core/components';
 import { styled } from '@storybook/core/theming';
-import { global } from '@storybook/global';
```

Comment:
This requires that line 32 below is then changed to

```
const { document } = globalThis;
```

(which is great, btw)

See the published Storybook failing to load https://635781f3500dd2c49e189caf-ytrwlnbeyv.chromatic.com/

Reply:
Sorry, looks like there are still some `global` references left, I missed those.

https://github.com/storybookjs/storybook/blob/6ad7a0b4bc45830065fa1aa1139527d0e8bc46fe/code/core/src/manager/components/sidebar/Search.tsx#L174-L175